### PR TITLE
[RDY] Added possibility to cut rebar cages with hacksaw or oxytorch

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Added rebar cage to iuse. I didn't add the rebar roof in this PR and I won't.

### CODE REVIEW

**Comment by kevingranade** [comment]
Need to try out your additions before PRing without a [WIP] or [CR] >:(
1. Doesn't compile, the ter_id stuff isn't as straightforward as it looks.  A simple way to make this work is replace t_reb_cage in iuse.cpp with ter_id( "t_reb_cage" ).
2. Doesn't work once that has been fixed, it tries to cut, but once it finishes it doesn't know how to transform a rebar cage, that code is at https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/activity_handlers.cpp#L2043

**Comment by BorkBorkGoesTheCode** [comment]
@kevingranade 

>Need to try out your additions before PRing without a [WIP] or [CR] >:(

I am sorry, I've done so now.

>Doesn't compile, the ter_id stuff isn't as straightforward as it looks. A simple way to make this work is replace t_reb_cage in iuse.cpp with ter_id( "t_reb_cage" ).

I've added 
`    t_reb_cage = ter_id( "t_reb_cage" );`

to void set_ters_ids in my offline copy, and I'm finishing the other changes. I'll upload after writing the commit messages.

File: src/activity_handlers.cpp [hunk 1]
```diff
@@ -1530,6 +1530,10 @@ void activity_handlers::oxytorch_finish( player_activity *act, player *p )
         g->m.ter_set( pos, t_window_empty );
         g->m.spawn_item( pos, "steel_plate", rng(0, 1) );
         g->m.spawn_item( pos, "sheet_metal", rng(1, 3) );
+    } else if( ter == t_reb_cage ) {
+        g->m.ter_set( pos, t_pit );
+        g->m.spawn_item( pos, "spike", rng(1, 19) );
+        g->m.spawn_item( pos, "scrap", rng(1, 8) );
```
Comment (Night-Pryanik) [inline src/activity_handlers.cpp]: Astyle these two lines, and similar ones in hacksaw below.

Comment (BorkBorkGoesTheCode) [reply src/activity_handlers.cpp]: @Night-Pryanik I'm figuring out how to use the Astyle binary 3.1 that came with Astyle source. The silent failures are a weird experience. Will upload after I figure it out. At least there are manuals.

Comment (BorkBorkGoesTheCode) [reply src/activity_handlers.cpp]: @Night-Pryanik honestly that would waste time and money.

**Comment by kevingranade** [comment]
Why closed? It looks like you adressed the issues.

**Comment by Night-Pryanik** [comment]
Renamed your PR so it properly describes what it changes.

**Pull request merged**